corehq/apps/registration/forms.py

```diff
@@ -493,18 +496,28 @@ def __init__(self, data=None, excluded_emails=None, is_add_user=None, location=N
         super(AdminInvitesUserForm, self).__init__(data=data, **kwargs)
         domain_obj = Domain.get_by_name(domain)
         self.fields['role'].choices = role_choices
-        if domain_obj.commtrack_enabled:
+        if domain_obj:
             self.fields['location_id'] = forms.CharField(label='Primary Location', required=False,
-                                                          widget=LocationSelectWidget(domain_obj.name),
-                                                          help_text=EMWF.location_search_help,
-                                                          initial=location.location_id if location else '')
-            self.fields['program'] = forms.ChoiceField(label="Program", choices=(), required=False)
-            programs = Program.by_domain(domain_obj.name)
-            choices = list((prog.get_id, prog.name) for prog in programs)
-            choices.insert(0, ('', ''))
-            self.fields['program'].choices = choices
+                                                         widget=LocationSelectWidget(domain_obj.name,
+                                                                                    id='id_location_id'),
+                                                         help_text=EMWF.location_search_help)
+
+            if domain_has_privilege(domain_obj.name, privileges.APP_USER_PROFILES):
```

Comment:
I think it'd also be nice to only show this is there are profiles defined.

Reply:
[36ab85d](https://github.com/dimagi/commcare-hq/pull/34466/commits/36ab85df23b87dd6db30e21d3b648721794f0d34)